src/serverdlg.cpp

```diff
@@ -154,6 +154,13 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
 
     // Tab: options
 
+    // Interface Language
+    QString strWTLanguage = "<b>" + tr ( "Language" ) + ":</b> " + tr ( "Select the language to be used for the user interface." );
+    lblLanguage->setWhatsThis ( strWTLanguage );
+    cbxLanguage->setWhatsThis ( strWTLanguage );
+
+    cbxLanguage->setAccessibleName ( tr ( "Language combo box" ) );
```

Comment:
Is the term "combo box" generally understood by non-programmers who need accessibility?

Reply:
Given how most of the AccessibleName values are set, I think it's the standard we're adhering to.  The server ones seem a bit eratic but the client ones seem fairly strict about the AN specifying what the widget type is.  Whether that's correct in UI/UX/Accessibility terms, I don't know.

@gilgongo can you comment?

Reply:
OK, yes, I see a lot of others are labelled "xxx combo box" too, so that's fine. Any change would be a separate exercise.